src/runner/browser-env/vite/browser-modules/globals.ts

```diff
@@ -12,9 +12,7 @@ const connectToSocket = (): BrowserViteSocket => {
         transports: ["websocket"],
     }) as BrowserViteSocket;
 
-    socket.on("connect_error", err => {
-        document.body.insertAdjacentHTML("afterbegin", `<p>${err.message}</p>`);
-    });
```

Comment:
It was mistake. I shouldn't change the user's layout here. Now I write it only to the log.

Reply:
On the other hand, how would user know if `connect_error` occured in the CI with remote grid? It is hard to check browser console error logs after the run ends

Reply:
> On the other hand, how would user know if `connect_error` occured in the CI with remote grid? It is hard to check browser console error logs after the run ends

Adding an element with error to the page won't help much either. In one of the next PR I will add the ability to send logs from the browser to the server. This will help you to see such errors.

TODO: add `runUuid` to the error in order to be able to understand which test fell.